core/issuers.chainmail

```diff
@@ -21,21 +21,36 @@ interface Issuer (Amount (Quantity)) {
   /** Make an empty purse associated with this kind of right. */
   makeEmptyPurse(name :String) -> (Purse);
 
+  /** Combine multiple payments into one payment. */
+  combine(paymentsArray :List(Payment), name :String = 'combined
+  payment') -> (Payment)
+
+  /** 
+   * Split a single payment into multiple payments, according to the
+   * amounts and names passed in. 
+   */
+  split(payment, amountsArray :List(Amount), namesArray
+  :List(String)
+    -> (List(Payment))
```

Comment:
The return type should be outside the parens for the parameters.

`
 split(payment, amountsArray :List(Amount), namesArray  :List(String))
    -> (List(Payment)
`

Reply:
Good catch